index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/styles.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="#" class="logo"> My Logo </a>
+        <div class="menu">
+          <img src="images/svg/Menu.svg" alt="Menu" />
+        </div>
+        <ul class="nav-menu">
+          <li class="nav-item">
+            <a href="#" class="hover-blue-text"> Portfolio </a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="hover-blue-text"> About </a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="hover-blue-text"> Contact </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="header">
+      <div class="container">
+        <h1 class="title">
+          I'm Eduardo <br />
+          Glad to see you!
+        </h1>
+        <p class="description">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hesitate to contact
+          me.
+        </p>
+        <a href="#" class="connect-button hover-blue-text">Let's connect</a>
+        <div class="rrss">
+          <a href="twitter.com"
+            ><img src="images/svg/twitter.svg" alt="twitter" /></a
+          ><a href="linkedin.com"
+            ><img src="images/svg/Linkedin.svg" alt="LinkedIn" /></a
+          ><a href="https://medium.com"
+            ><img src="images/svg/medium.svg" alt="Medium" /></a
+          ><a href="github.com"
+            ><img src="images/svg/github.svg" alt="Github" /></a
+          ><a href="angellist.com"
+            ><img src="images/svg/angellist.svg" alt="Angel List"
+          /></a>
+        </div>
```

Comment:
- [ ] It is important to make use of semantic tags as much as possible in our application and as such i will suggest rather than using the non-semantic `div` tag as container for the social media icons, you make use of the semantic `ul` tag and each link would be wrapped in an `li` tag as a list item. see the example below for more reference :+1: 
```html
<ul class="socials">
  <li>
    <a href="https://github.com"><img src="./assets/github.png" alt="github"/></a>
  </li>
  <li>
    <a href="https://linkedin.com/in"><img src="./assets/linkedin.png" alt="linkedin"/></a>
  </li>
</ul>
```